core/vehicle/api.go

```diff
@@ -43,6 +43,13 @@ type API interface {
 	// SetPlanSoc sets the charge plan time and soc
 	SetPlanSoc(time.Time, int) error
 
+	// GetRepeatingPlans returns every repeating plan
+	GetRepeatingPlans(onlyActivePlans bool) []api.RepeatingPlanStruct
```

Comment:
Den `onlyActivePlans` Filter könnten wir auch rauslassen und erst in der Listenbearbeitung wo das für uns relevant ist prüfen. Ist aktuell ja nur die eine Stelle, wenn ich das richtig sehe.